wls-gui-wahllokalsystem/tests/composables/wahlvorstand/wahlvorstandservice.spec.ts

```diff
@@ -73,7 +73,9 @@ describe("WahlvorstandService.ts", () => {
           Promise.resolve({ data: createWahlvorstandDTO() })
         );
 
-        await unitUnderTest.getWahlvorstand(wahlbezirkID, forceUpdate);
+        await unitUnderTest.getWahlvorstand(wahlbezirkID, {
+          forceUpdate: forceUpdate,
+        });
```

Comment:
Um auch zu prüfen dass der Default von `sendNotification` verwendet wurde

```suggestion
        await unitUnderTest.getWahlvorstand(wahlbezirkID, {
          forceUpdate: forceUpdate,
        });
        expect(mockDefinitions.addNotification.mock.calls).toEqual([
          [expect.any(String), UserNotificationCategoryEnum.SUCCESS],
        ]);
```

Reply:
[88c9b77](https://github.com/it-at-m/Wahllokalsystem/pull/1399/commits/88c9b77e37fe052ec36e81054d605343d9aa0c18)